bulkNominatim.py

```diff
@@ -68,18 +68,18 @@ def unsetTool(self, tool):
     def unload(self):
         """Unload BulkNominatim from the QGIS interface."""
         self.canvas.unsetMapTool(self.reverseGeocodeTool)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.nominatimAction)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.reverseGeocodeAction)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.settingsAction)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.helpAction)
+        self.iface.removePluginMenu(u"LocationIQ GeoCoding", self.nominatimAction)
+        self.iface.removePluginMenu(u"LocationIQ GeoCoding", self.reverseGeocodeAction)
+        self.iface.removePluginMenu(u"LocationIQ GeoCoding", self.settingsAction)
```

Comment:
there are 2 setting menus added to the UI, one's a dummy

Reply:
I am sorry I don't get what you mean by one's a dummy?